pkgs/development/tools/electron/default.nix

```diff
@@ -36,7 +36,16 @@ let
       };
     }.${stdenv.hostPlatform.system} or throwSystem;
 
-    buildInputs = [ unzip makeWrapper ];
+    buildInputs = [
+      unzip
+      makeWrapper
+      gtk3
```

Comment:
Everything except gtk should go to `nativeBuildInputs` as they are run at build time.

Reply:
I guess not the unzip too? 
```
replacing old 'electron-4.0.0'
installing 'electron-4.0.0'
these derivations will be built:
  /nix/store/yqdi8p6vw6jw046mrqi17bw9zh0k6asc-electron-4.0.0.drv
building '/nix/store/yqdi8p6vw6jw046mrqi17bw9zh0k6asc-electron-4.0.0.drv'...
/nix/store/yax21ms90cwr2l4rygd60wfbpfwg5hip-stdenv-linux/setup: line 1272: unzip: command not found
builder for '/nix/store/yqdi8p6vw6jw046mrqi17bw9zh0k6asc-electron-4.0.0.drv' failed with exit code 127                                                                                                   
error: build of '/nix/store/yqdi8p6vw6jw046mrqi17bw9zh0k6asc-electron-4.0.0.drv' failed
```

Reply:
my bad....

Reply:
done